config.default.yaml

```diff
@@ -91,6 +91,7 @@ electricity:
   co2limit: 7.75e+7  # European default, 0.05 * 3.1e9*0.5, needs to be adjusted for Africa
   co2base: 1.487e9  # European default, adjustment to Africa necessary
   agg_p_nom_limits: data/agg_p_nom_minmax.csv
+  hvdc_as_lines: true
```

Comment:
Please add a description of the setting

Reply:
This applies also to the other config*.yaml files

Reply:
@ekatef I think this comment was missing, all the rest seems to be already covered :)